src/Forms/Schema/FormSchema.php

```diff
@@ -2,10 +2,10 @@
 
 namespace SilverStripe\Forms\Schema;
 
-use SilverStripe\Control\Session;
 use SilverStripe\Forms\CompositeField;
 use SilverStripe\Forms\Form;
 use SilverStripe\Forms\FormField;
+use SilverStripe\ORM\ValidationResult;
 
 /**
  * Represents a {@link Form} as structured data which allows a frontend library to render it.
```

Comment:
This comment isn't really correct. FormSchema doesn't represent a form, because it doesn't any state. It's more like a service/factory that builds JSON representations of a Form Schema on demand. It's not related to this PR but it confused me a bit.